modules/platform/nuxeo-platform-audit/nuxeo-platform-audit-core/src/test/java/org/nuxeo/ecm/platform/audit/io/LogEntryJsonWriterTest.java

```diff
@@ -41,6 +44,8 @@
 import org.nuxeo.runtime.test.runner.Deploy;
 import org.nuxeo.runtime.test.runner.Features;
 
+import static org.nuxeo.common.utils.DateUtils.formatISODateTime;
```

Comment:
The formatter should put static imports before standard ones

Reply:
done